plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
+    """
+
+    @staticmethod
+    def nm_msg():
+        return "I'm sorry :("    
```

Comment:
If we are doing everything through the static methods, then its perfectly fine to only test them.

Also, testing both the command and the helper method isn't worth it imo.